lib/app/service/nekoton_related/connection_service/connection_service.dart

```diff
@@ -42,18 +42,16 @@ class ConnectionService {
   /// by its type and put it in nekoton.
   Future<void> _updateTransportByConnection(ConnectionData connection) async {
     _log.finest('updateTransportByConnection: ${connection.name}');
-    final type = connection.networkType;
-
     final transport = await connection.when<Future<Transport>>(
-      gql: (_, name, networkId, group, endpoints, timeout, __, ___) =>
+      gql: (_, name, networkId, group, endpoints, timeout, __, isLocal, ___) =>
```

Comment:
For the future: i think we must add here `ticker`, `currencyUrl`, `icon` and maybe something else for all connections, but for ever/venom it will be constants or just empty values

Reply:
Yes, but we need to decide on the data source for the remaining fields of TransportStrategy for custom network.

Reply:
I thought it should be set on `ConfigurePage`

Reply:
All of them? :)

Reply:
looks like, if user not specified it, we should use some default values, I don't see another option

Reply:
But they are NOT present in browser extension (network id, group etc) as user-definable fields.

Reply:
Agrh... Idk what to do. Let's leave it now